docs/api/result.md

```diff
@@ -1,12 +1,16 @@
+---
+anta_title: ANTA Result API
+---
 <!--
   ~ Copyright (c) 2023-2025 Arista Networks, Inc.
   ~ Use of this source code is governed by the Apache License 2.0
   ~ that can be found in the LICENSE file.
   -->
 
-# Test Result model
+## ::: anta.result_manager.ResultManager
```

Comment:
wondering if the docstring fo ResultManager is still correct in the example (if not maybe we fix it in this doc PR).

Reply:
Is it worth maintaining that? That example is not a real life use case.
We would use the ANTA runner.